### PR TITLE
14-vm: Fix minor issues related to grammar, topic, typos and style

### PR DESCRIPTION
**Pull request by elena19m**
I have added minor fixes for chapter 14: Mașini Virtuale. The changes are related to typos, grammar, phrase topic or incoherency, wrong caption for figures (they were not in concordance with the content of the figures). 
Some of the changes (e.g. `\newpage`) were necessary to pretty print the text in the PDF file (e.g. figures were displayed in wrong sections). Please let me know if they break the rules and I will remove them.

### CODE REVIEW

File: book/chapters/14-vm/contents.tex [hunk 1]
```diff
@@ -562,43 +583,43 @@ \section{Anexă: Crearea unei mașini virtuale în VirtualBox}
 		spațiu disponibil pe discul fizic.
 \end{itemize}
 
-În ultimă fază trebuie să specificați dimensiunea pe care o doriți pentru disc.
+În ultimă fază, trebuie să specificați dimensiunea pe care o doriți pentru disc.
```
Comment (razvand) [inline book/chapters/14-vm/contents.tex:586]:
```suggestion
În ultimă fază, trebuie să indicați dimensiunea pe care o doriți pentru disc.
```

File: book/chapters/14-vm/contents.tex [hunk 2]
```diff
@@ -562,43 +583,43 @@ \section{Anexă: Crearea unei mașini virtuale în VirtualBox}
...
 
 \begin{itemize}
-  \item \texttt{System} - putem configura cantitatea de memoriei, numărul de
-		procesoare
-              \item \texttt{Display} - cantitatea de memorie utilizată de placa video
-              \item \texttt{Storage} - putem vedea ce discuri sunt atașate, ce proprietăți au.
+  \item \texttt{System} - cantitatea de memoriei, numărul de
+		procesoare.
+  \item \texttt{Display} - cantitatea de memorie utilizată de placa video.
+  \item \texttt{Storage} - discurile ce sunt atașate, precum și ce proprietăți au.
```
Comment (razvand) [inline book/chapters/14-vm/contents.tex:609]:
```suggestion
  \item \texttt{Storage} - discurile atașate și proprietățile lor.
```

File: book/chapters/14-vm/contents.tex [hunk 3]
```diff
@@ -562,43 +583,43 @@ \section{Anexă: Crearea unei mașini virtuale în VirtualBox}
...
 		Tot aici putem adăuga și un disc virtual necesar instalării
 		mașinii virtuale. În \labelindexref{Figura}{fig:vm:vbox-iso}
 		este reprezentat modul prin care puteți selecta o imagine de tip
-		ISO pentru a fi montată ca și un disc virtual în mașina virtuală
+		ISO pentru a fi montată ca un disc virtual în mașina virtuală
 		cu scopul de a instala sistemul de operare.
-              \item \texttt{Audio} - activare/dezactivare a plăcii audio pentru mașina virtuală
-              \item \texttt{Network} - configurarea plăcii/plăcilor de rețea pe care mașina
+  \item \texttt{Audio} - activarea/dezactivarea plăcii audio pentru mașina virtuală.
```
Comment (razvand) [inline book/chapters/14-vm/contents.tex:615]:
```suggestion
  \item \texttt{Audio} - activarea / dezactivarea plăcii audio pentru mașina virtuală.
```

File: book/chapters/14-vm/contents.tex [hunk 4]
```diff
@@ -562,43 +583,43 @@ \section{Anexă: Crearea unei mașini virtuale în VirtualBox}
...
+  \item \texttt{Network} - configurarea plăcii/plăcilor de rețea pe care mașina
```
Comment (razvand) [inline book/chapters/14-vm/contents.tex:616]:
```suggestion
  \item \texttt{Network} - configurarea plăcii / plăcilor de rețea pe care mașina
```

**Review by razvand — APPROVED** [review]
Arată foarte bine. Mulțumesc. Am adăugat câteva sugestii.

**Comment by elena19m** [comment]
I have applied the suggested changes for Chapter 14: Mașini virtuale.

File: book/chapters/14-vm/contents.tex [hunk 5]
```diff
@@ -562,43 +583,43 @@ \section{Anexă: Crearea unei mașini virtuale în VirtualBox}
...
 \end{figure}
 
 Printre cele mai relevante opțiuni de configurare sunt:
 
 \begin{itemize}
-  \item \texttt{System} - putem configura cantitatea de memoriei, numărul de
-		procesoare
-              \item \texttt{Display} - cantitatea de memorie utilizată de placa video
-              \item \texttt{Storage} - putem vedea ce discuri sunt atașate, ce proprietăți au.
+  \item \texttt{System} - cantitatea de memoriei, numărul de
```
Comment (razvand) [inline book/chapters/14-vm/contents.tex:606]:
```suggestion
  \item \texttt{System} - cantitatea de memorie, numărul de
```

File: book/chapters/14-vm/contents.tex [hunk 6]
```diff
@@ -562,43 +583,43 @@ \section{Anexă: Crearea unei mașini virtuale în VirtualBox}
...
+		procesoare.
+  \item \texttt{Display} - cantitatea de memorie utilizată de placa video.
+  \item \texttt{Storage} - discurile atașate și ce proprietăți lor.
```
Comment (razvand) [inline book/chapters/14-vm/contents.tex:609]:
```suggestion
  \item \texttt{Storage} - discurile atașate și proprietățile lor.
```

**Review by razvand — APPROVED** [review]
Am lăsat două sugestii. După aceasta, putem să facem merge.

**Comment by elena19m** [comment]
Am aplicat runda a 2-a de sugestii pentru 14-vm: Masini Virtuale.

**Pull request merged**